src/main/java/seedu/programmer/logic/parser/ParserUtil.java

```diff
@@ -111,6 +111,8 @@ public static int parseLabNum(String labNum) throws ParseException {
             int value = Integer.parseInt(labNum);
             if (value <= 0) {
                 throw new ParseException(Lab.MESSAGE_LAB_NUMBER_CONSTRAINT);
+            } else if (value > 14) {
+                throw new ParseException(Lab.MESSAGE_LAB_NUMBER_CONSTRAINT);
```

Comment:
It seems the same exceptions are thrown in both cases here. Could we combine both conditions with an || operator?